metadata-ingestion/tests/integration/metabase/test_metabase.py

```diff
@@ -14,7 +14,8 @@
 JSON_RESPONSE_MAP = {
     "http://localhost:3000/api/session": "session.json",
     "http://localhost:3000/api/user/current": "user.json",
-    "http://localhost:3000/api/dashboard": "dashboard.json",
```

Comment:
can we delete the `dashboard.json` file? or is that still used elsewhere in the tests?

Reply:
I struggled to run these tests, actually. I did set my machine to java 8 but couldn't run the gradlew successfully.
I'll put some more thought to it soon.

Reply:
@hsheth2 I'm using my GH Actions to test it https://github.com/7onn/datahub/actions
But I couldn't find how to run these python tests nor run the gradlew from this mac.

Reply:
We've actually migrated to java 17 now

Have you taken a look at these docs https://datahubproject.io/docs/next/metadata-ingestion/developing/

Let me know if you run into issues with it - we want developing on ingestion, including tests, to be a smooth experience